charts/covidDataExplorer/CovidDataExplorer.tsx

```diff
@@ -88,6 +91,25 @@ export class CovidDataExplorer extends React.Component<{
         )
     }
 
+    static async redirectAndBootstrap(
```

Comment:
Nit: This method is just doing a replaceState right? Maybe better to not have `redirect` in the name?

Reply:
Yeah it's a bad name for what it does. Should it be `replaceStateAndBootstrap` maybe?